docs/conf.py

```diff
@@ -145,7 +147,7 @@
 
 # The name for this set of Sphinx documents.
 # "<project> v<release> documentation" by default.
-#html_title = u'rasa_nlu v0.3.0'
+html_title = u''
```

Comment:
should we really set this to the empty string?

Reply:
yeah that's what we want. This is the string that gets appended to your existing title, which makes it really long for no reason